dependency-check-suppressions.xml

```diff
@@ -55,6 +55,7 @@
         <cve>CVE-2017-18640</cve>
         <cve>CVE-2019-17638</cve>
         <cve>CVE-2020-24616</cve>
+        <cve>CVE-2020-5421</cve>
```

Comment:
Add a note mentioning why we are suppressing this CVE

Reply:
@AyanDuttaHMCTS - I am trying to fix it. Hopefully i can remove this CVE. If not i will definitely add a note.

Reply:
Added notes